src/settings_gui.cpp

```diff
@@ -417,6 +417,8 @@ struct GameOptionsWindow : Window {
 	int warn_lines = 0; ///< Number of lines used for warning about missing search results.
 
 	Scrollbar *vscroll;
+	Scrollbar *vscroll_description;
```

Comment:
Strictly this (and all the other `vscroll`) probably isn't necessary, as `this->GetScrollbar(...)` isn't particuarly expensive.

Reply:
True, we do this everywhere. Funnily `BuildBridgeWindow` assigns it, but never uses is.
Maybe it was needed in the past. Some other PR :)